src/script/components/panel/ConversationProtocolDetails/ConversationProtocolDetails.tsx

```diff
@@ -22,38 +22,9 @@ import React from 'react';
 import {CSSObject} from '@emotion/react';
 import {ConversationProtocol} from '@wireapp/api-client/lib/conversation/NewConversation';
 
-import {t} from 'Util/LocalizerUtil';
+import {Ciphersuite} from '@wireapp/core-crypto';
```

Comment:
hummm this doesn't sound safe. CoreCrypto is lazy loaded and in some case it might not even be loaded. 

Also the webapp should not even be aware that coreCrypto exists. 
Maybe the `core` could expose such enum, WDYT?

Reply:
Ok, but then should we reexport corecrypto's enum from core, or redefine similar enum ourselfes? 🤔